docs/CONTRIBUTING.md

```diff
@@ -29,7 +29,8 @@ An easier solution would be to use Docker.
 **Setup**
 1. Navigate to the root directory of telescope.
 1. Copy env.example to .env to create a new environment configuration.
-1. Replace default key values in .env with credentials. 
+1. Replace default key values in .env with credentials.
+1. Begin the Redis container using `docker-compose up -d`
```

Comment:
This actually initializes both Redis and telescope. For local testing, it's better to just execute `redis-server` in a terminal and then following the next steps (`npm install`, `npm test`...)

Reply:
Nice catch!! Running `docker ps` showed both the container for telescope and redis running - I think we'd only need redis running for the tests.

I'm not sure I follow your suggestion to run the command, `redis-server`. From my understanding, the docker container for Redis has been made available such that a contributor doesn't need to have Redis installed on their machine - they can run the container, whether they're on Windows/Mac/Linux or some other variant, and this will remove the dependency of having Redis installed locally.

Please let me know if anything is unclear! 😀

Reply:
The idea was to run everything as a [multi-container Docker application](https://docs.docker.com/compose/), but now that you mentioned it, I think what you suggested is a good idea: available such that contributors don't need to have Redis installed on their machine.

For that, we can do this.
- First time a contributor runs a docker container for redis:
`docker run --name redis-telescope -p 6379:6379 redis`
This creates a container for redis named redis-telescope and exposes it on port 6379, the one we need for telescope.

- Contributor wants to run the same container again:
`docker start -a redis-telescope`
This runs the same container, using the option `-a` (foreground running) so the contributor can see the output.
For running it in detached mode, don't use `-a`.

For removing the container redis container: `docker container rm redis-telescope`.

@PagerDuty test it and let me know what you think.

CC: @Reza-Rajabi What do you think about all this?

Reply:
What you're saying is making total sense! I have a few questions:

* I think we might want to reconsider using `docker run --name redis-telescope -p 6379:6379 redis` over `docker-compose up -d redis`. The Redis container metadata could possibly change in the future and having it specified in the docker-compose file ensures that we have a single source of truth - what do you think? 🤔

* I didn't see those commands you suggested for spinning up and taking down the Redis container anywhere in the `CONTRIBUTING` file nor the `README`. I want to try and avoid increasing the scope/complexity in this patch but I think it would definitely be worth a discussion for another PR!

I think to produce a frictionless experience for newcomers so they don't get frustrated that they can't even setup the project, we should keep it simple. See below for the entire overview of what I'm suggesting:

![image](https://user-images.githubusercontent.com/13009507/68723000-4c78dd00-0585-11ea-94c3-362676937867.png)

Reply:
Oh! My bad, you're right, going with `docker-compose up -d redis` is definitely better. Just one thing, if you want to recommend running in detached mode `-d`, maybe you could mention how to stop it. Other than that, I think it's good to go.